### PR TITLE
Implement Container methods

### PR DESCRIPTION
**Pull request by Limeth**
[SpongeCommon](https://github.com/SpongePowered/SpongeCommon/pull/1209/) | [SpongeAPI](https://github.com/SpongePowered/SpongeAPI/pull/1491)

### CODE REVIEW

**Comment by liach** [comment]
Can you fix the merge conflict?

**Comment by parlough** [comment]
Will close these pull requests in 15 days or so if not updated for the latest changes due to inactivity.

File: src/main/java/org/spongepowered/api/item/inventory/Container.java [hunk 1]
```diff
@@ -57,17 +57,24 @@
...
      * @param viewer The viewer to stop showing this inventory to
      * @param cause The {@link Cause} to provide when closing the inventory
-     * @throws IllegalArgumentException if a {@link PluginContainer} is not the root of the cause
+     * @return {@code true}, if the {@link Container} was closed, {@code false}
+     *         otherwise.
+     * @throws IllegalArgumentException if a {@link PluginContainer} is not the
+     *         root of the cause
      */
-    void close(Player viewer, Cause cause) throws IllegalArgumentException;;
+    boolean close(Player viewer, Cause cause) throws IllegalArgumentException;
```
Comment (ryantheleach) [inline src/main/java/org/spongepowered/api/item/inventory/Container.java:78]: The cause has been removed in later versions.

File: src/main/java/org/spongepowered/api/item/inventory/Container.java [hunk 2]
```diff
@@ -57,17 +57,24 @@
...
-     * Stops showing this Inventory to the given viewer.
+     * Stops showing this Inventory to the given viewer, if it is currently
+     * being shown.
      *
      * @param viewer The viewer to stop showing this inventory to
      * @param cause The {@link Cause} to provide when closing the inventory
-     * @throws IllegalArgumentException if a {@link PluginContainer} is not the root of the cause
+     * @return {@code true}, if the {@link Container} was closed, {@code false}
+     *         otherwise.
+     * @throws IllegalArgumentException if a {@link PluginContainer} is not the
```
Comment (ryantheleach) [inline src/main/java/org/spongepowered/api/item/inventory/Container.java:75]: If the cause is removed, the API methods should probably not throw an IllegalArgumentException any more right?

**Comment by parlough** [comment]
Going to close this due to inactivity, if you'd still like to get this in, update for the latest changes and open a new pull request.

Thanks for your contribution!

**Pull request closed without merging**